torch/_inductor/lowering.py

```diff
@@ -779,11 +779,15 @@ def repeat(x, repeats):
 
     new_size = list(x.get_size())
 
+    zero_tensor = False
     for i in range(len(repeats)):
-        assert repeats[i] != 0
+        if repeats[i] == 0:
+            zero_tensor = True
         if repeats[i] != 1:
             new_size[i] = new_size[i] * repeats[i]
```

Comment:
just curious: why is this check needed? When `repeats[i] == 1`, `new_size[i] = new_size[i] * repeats[i]` becomes a nop anyway?

Reply:
This check is unnecessary. Removed.